backend/api/urls.py

```diff
@@ -0,0 +1,17 @@
+from django.conf.urls import url
```

Comment:
Django 2.0+ provides better way for defining URLs. `django.urls.path` can be used instead of `django.conf.urls,url`. In this way, no regex is needed.

It is optional though.

Reply:
I think we can change it later, we're too busy with other things...